office/todotxt/Portfile

```diff
@@ -38,15 +38,17 @@ destroot {
 }
 
 post-destroot {
-    set completions_path ${destroot}${prefix}/share/bash-completion/completions
-    xinstall -d ${completions_path}
-    xinstall -m 644 ${worksrcpath}/todo_completion ${completions_path}/${name}
+    if {[variant_isset bash_completion]} {
+        set completions_path ${destroot}${prefix}/share/bash-completion/completions
+        xinstall -d ${completions_path}
+        xinstall -m 644 ${worksrcpath}/todo_completion ${completions_path}/${name}
+    }
```

Comment:
Okay, so, as I suggested in the other comments, I don't think we need a variant for this, and I'd suggest you reduce this PR to just fixing the reinplace.